1-CONTRIBUTION-GUIDE/README.md

```diff
@@ -83,9 +83,12 @@ To make sure your template is added to Azure.com index, please follow these guid
 
 + Guidelines on the README.md file below.
 
-1. The deployment template file must be named **azuredeploy.json**.
+1. The bicep deployment file (optional) must be named **main.bicep**
+1. The JSON deployment template file (required) must be named **azuredeploy.json**.
 1. There should be a parameters file named **azuredeploy.parameters.json**.
```

Comment:
I think this is a *must* (I know it's not in your CL)

Reply:
I don't follow.  This is taken care of by the 2nd item:

>> 2. The JSON deployment template file (required) must be named **azuredeploy.json**.